src/main/proto/admin.proto

```diff
@@ -290,6 +290,9 @@ message EventProcessorIdentifier {
 
   /* the identifier of the token store this event processor is using */
   string token_store_identifier = 2;
+
+  /* the context name where the processor is located */
```

Comment:
add comment that this field is optional and if not specified the context for the channel will be used.